packages/odyssey-react/src/components/Table/TableCell.theme.ts

```diff
@@ -0,0 +1,35 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { ThemeReducer } from "@okta/odyssey-react-theme";
+
+export const theme: ThemeReducer = (theme) => ({
+  MaxLineLength: theme.FontLineLengthMax,
+
+  BorderWidth: theme.BorderWidthBase,
+  BorderStyle: theme.BorderStyleBase,
+  BorderColor: theme.ColorBorderDisplay,
+  PaddingBlock: theme.SpaceRemS,
+  PaddingInline: theme.SpaceRemS,
+
+  HeaderColor: theme.ColorTextBody,
+  HeaderWeight: theme.FontWeightBold,
+
+  HeaderColBorderWidth: theme.BorderWidthBase,
+  HeaderColBorderStyle: theme.BorderStyleBase,
+  HeaderColBorderColor: theme.ColorBorderDisplay,
+
+  EmptyPaddingBlock: theme.SpaceRemL,
+  EmptyPaddingInline: theme.SpaceRemXl,
+  EmptyBackgroundColor: theme.ColorBackgroundDisabled,
+  EmptyColor: theme.ColorTextSub,
```

Comment:
one of the things I like about adding these name "parts" as a suffix (as opposed to a prefix here) is that they occupy the same part of the variable name that variants do - so all the tokens become more uniform and consistent across the API!